thriftpy/thrift.py

```diff
@@ -176,6 +179,7 @@ def _recv(self, _api):
 
 
 class TProcessor(object):
+
```

Comment:
We use flake8 for coding style checking. And this kind of blank lines are not forced to add in flake8. So I think we should better not add the blank lines here and it's not consistent with other class definitions.